stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -1,4 +1,8 @@
-apiVersion: extensions/v1beta1
+{{- if semverCompare "^1.9-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
you can use https://github.com/helm/helm/blob/master/docs/chart_template_guide/builtin_objects.md apiVersion.has directly instead

Reply:
oooh, good call, I'll test it out and push the update. Thanks for this!